flask_resources/loaders.py

```diff
@@ -7,11 +7,36 @@
 
 """Loader module."""
 
+from functools import wraps
+
 from flask import request
+from werkzeug.exceptions import UnsupportedMediaType
 
 from .context import resource_requestctx
 
 
+def request_loader(f):
+    """Decorator that sets the request_loader on the view."""
+
+    @wraps(f)
+    def wrapper(self, *args, **kwargs):
+        """Wrapping method.
+
+        :params self: ItemView or ListView class
+        """
+        # Checking Content-Type is the responsibility of the deserializer/loader
```

Comment:
minor: loader, deserializer only takes a known (for it) format and spits out another known format. Should not know anything about request. Code is good, just comment:

```suggestion
        # Checking Content-Type is the responsibility of the loader
```

Reply:
Makes sense